docs/csharp/package-management.md

```diff
@@ -27,3 +27,17 @@ For projects that use `<PackageReference>`, you can see the package references i
 Packages that don't install properly when a restore happens or you run a build will show error icons in **Solution Explorer**.
 
 **Note**: At this time, you cannot right-click on projects to manage your NuGet packages and there is not a NuGet Package Manager user interface in Visual Studio Code. For more information on managing packages, see [Install and manage NuGet packages with the dotnet CLI](https://learn.microsoft.com/nuget/consume-packages/install-use-packages-dotnet-cli).
+
+### Project-to-project references
+
+Project-to-project references may be added via the Solution Explorer UI. You can invoke the "Add Project Reference" command by right clicking on a project node in the Solution Explorer and selecting the command in the context menu.
```

Comment:
The convention on code.visualstudio.com is to use bold for UI text and command names so **Add Project Reference**